planning/behavior_path_planner/include/behavior_path_planner/scene_module/scene_module_interface.hpp

```diff
@@ -503,7 +508,7 @@ class SceneModuleInterface
   PlanResult path_candidate_;
   PlanResult path_reference_;
 
-  ModuleStatus current_state_;
+  ModuleStatus current_state_{ModuleStatus::RUNNING};
```

Comment:
Any reason to set RUNNING here?
The initial value should be `IDLE` for the new architecture and `SUCCESS` for the old one.

https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/include/behavior_path_planner/scene_module/scene_module_interface.hpp#L71-L75

Reply:
There is no particular reasons.
The value required to be initialized when tick() is called.
Or else, the planning module will not run for old archi. I can try  set and run `IDLE` and `RUNNING` once.

Reply:
fixed in ddd11de07387895cad3a6efe3f700bbb4ba3bb93